src/components.d.ts

```diff
@@ -42,23 +43,30 @@ export namespace Components {
     }
     interface PageTrucks {
     }
+    interface UiAddressInput {
+        "buttonLabel": string;
+        "getFormattedAddress": () => Promise<string>;
+        "label": string;
+        "name": string;
+        "placeholder": string;
+    }
     interface UiCard {
-        "headerText": string;
+        "headerText"?: string;
         "isActive": boolean;
         "isCollapsible": boolean;
         "isSmall": boolean;
         /**
           * Set an `id` on the card element to allow navigating or scrolling to it
          */
-        "scrollId": string;
+        "scrollId"?: string;
     }
     interface UiDynamicText {
         "format"?: (value: any) => string;
         "value": any | undefined;
     }
     interface UiGeoMap {
         "getCenter": () => Promise<google.maps.LatLng | undefined>;
-        "setCenter": (newCenter: google.maps.LatLngLiteral) => Promise<void>;
+        "setCenter": (newCenter: google.maps.LatLngLiteral, showMarker?: boolean) => Promise<void>;
```

Comment:
map doesn't seem to update

![image](https://user-images.githubusercontent.com/700444/97742997-4172d180-1aa2-11eb-972d-aea6d80776ee.png)

Reply:
I'll take a look, it should work if the address is correct, but there is a similar issue of what to do when the address returns 404.